internal/datastore/mysql/datastore.go

```diff
@@ -491,6 +498,20 @@ func (mds *Datastore) seedDatabase(ctx context.Context) error {
 		return fmt.Errorf("seedDatabase: failed to get last inserted id: %w", err)
 	}
 
+	uuidSQL, uuidArgs, err := sb.
```

Comment:
if multiple SpiceDB processes attempt to do this there will be most likely a race, and multiple rows will be inserted. You could solve this by adding an ID column and inserting on a known ID (e.g. zero), and use `INSERT IGNORE`. That's the strategy used with the zeroth transaction.